app/routes.py

```diff
@@ -12,7 +16,11 @@ def index():
         "header": [
             navbar_component(),
         ],
-        "main": [],
+        "main": [
+            best_bets_component(),
+            restaurant_card_component(),
+            restaurant_rating_component()
+        ],
```

Comment:
Why are these changes in this PR? these components are not related to the orders page.